src/module/timedEffect/timedEffect.js

```diff
@@ -108,19 +107,22 @@ export default class SFRPGTimedEffect {
             actor.system.timedEffects.find(tEffect => tEffect.id === this.id)?.update(this);
             game.sfrpg.timedEffects.find(gEffect => gEffect.id === this.id)?.update(this);
 
-            actor.update({'items': items});
+            actor.updateEmbeddedDocuments({'Item': effect});
```

Comment:
The arguments for `updateEmbeddedDocuments` are different to `update`. They're:
```ts
updateEmbeddedDocuments(embeddedDocumentType, diff[])
```
`embeddedDocumentType` is `"Item"` in this case.
`diff[]` is an array of objects, each object representing changes to an embedded document, and containing the item's `_id`. So you'll end up with something like 
```js
updateEmbeddedDocuments("Item", [
{
  _id: "123456...", 
  system: {
    someProperty: 1000
  }
}
])
```
You can pass full items into the diff array, and Foundry will calculate and apply the diff based on that new item, but creating the diff object yourself will save it from having to do that, and likely be more performant